src/Tuura/Concept/FSM/Translation.hs

```diff
@@ -247,14 +297,14 @@ addMissingSignals x = zip
     transitionList =  map fullList
     missingSignals y = map (getAllSignals y \\) (onlySignals y)
 
--- Take a list of transitions and convert it to a Tristate format containing
--- TriTrue, TriFalse and TriX (or don't care) states.
+-- | Take a list of transitions and convert it to a Tristate format containing
+-- TriTrue, TriFalse and TriX (or "don't care") states.
```

Comment:
Single and double quotes have a special meaning in Haddock. Perhaps, use `/don't care/` i.e. italics.